packages/ui/react-ui-theme/src/styles/components/alert.ts

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { type Theme, type ComponentFunction } from '@dxos/react-ui-types';
+
+import { mx } from '../../util';
+import { descriptionMessage } from '../fragments';
+
+export type AlertStyleProps = {};
+
+export const alertRoot: ComponentFunction<AlertStyleProps> = (_, etc) => {
+  return mx('max-w-[calc(100%-2rem)] is-full m-4 place-self-center text-center rounded-md', descriptionMessage, etc);
```

Comment:
Why subtract `2rem`? Why `m-4`? Why should root have `text-center` and not `Alert.TItle` or the content?

Reply:
If this intends to *force* a specific spacing around the element, it should wrap everything in a `div role='none'` which applies internal spacing, rather than trying to be robust to whatever display mode the host might be applying.